### DISCUSSION TITLE
Error: Could not create PCH

### DISCUSSION THREAD

**Discussion by mateus-x02** [HaxeFlixel/flixel#2599] [Q&A]
I'm building a Friday Night Funkin' mod with a workflow on Github, after all, an error appears.

Error: Could not create PCH.

The repository:
https://github.com/MateuzinhoX02/PsychEngine0.4.2-Android
The build log:
https://github.com/MateuzinhoX02/PsychEngine0.4.2-Android/actions/runs/2656833905

![image](https://user-images.githubusercontent.com/105178173/178516122-06c06d24-1579-41ee-9e8c-100f724879d7.png)

**Comment by mateus-x02** [comment]
Sorry about my English mistakes, I'm not fluent.

**Comment by RichardBray** [comment]
Does the code work when you run in locally?

**Reply by mateus-x02** [reply]
Note: I appreciate your understanding of trying to help me, but I already solved it! My workflow was out of date, I updated it, and it worked! I thank!
